lib/hanami/commands/generate/app.rb

```diff
@@ -7,11 +7,14 @@ module Commands
     class Generate
       class App < Abstract
 
+        DEFAULT_TEMPLATE_FORMATS = 'erb'.freeze
+
+        APPLICATION_TEMPLATE_FORMATS = %w[erb slim haml].freeze
```

Comment:
I think the code should be dynamic but we could have tests to cover all supported template engines